src/Parser/Druid/Balance/Modules/Spells/MoonSpells.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import { formatPercentage } from 'common/format';
+import Analyzer from 'Parser/Core/Analyzer';
+import SPELLS from 'common/SPELLS';
+
+class MoonSpells extends Analyzer {
+
+  get availableCasts() {
+    //const hasRM = combatant.hasBack(ITEMS.RADIANT_MOONLIGHT.id);
+    const hasRM = false;
+    const cooldown = hasRM ? 10 : 15;
```

Comment:
I suppose you're going to extend it in the future (judging from the comment)? If so, fine, but if not, `cooldown` should be directly 15 seconds.